api/main/config/db.js

```diff
@@ -22,7 +22,23 @@ db.sequelize = sequelize
 
 db.patient = require('../models/Patient.js')(sequelize, Sequelize)
 db.sourceFile = require('../models/SourceFile.js')(sequelize, Sequelize)
+db.configuration = require('../models/Configuration.js')(sequelize, Sequelize)
```

Comment:
not a huge fan of `sequelize` and `Sequelize`. Why do you need to pass this in as an arg when you could import `Sequelize` in the criteria definition files?